pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -286,12 +286,22 @@ func getProxyEnv(cfg *config.Cluster, networkName string, nodeNames []string) (m
 
 		noProxyList := append(subnets, envs[common.NOProxy])
 		noProxyList = append(noProxyList, nodeNames...)
```

Comment:
aren't we already adding these here?

Reply:
Good point! Indeed it looks like we can revert for docker and try to pass the list the same way as it's done here indeed.. I'll do it right away. Thanks!